db/migrate/20221124194524_create_transactional_outbox_outbox.rb

```diff
@@ -0,0 +1,13 @@
+class CreateTransactionalOutboxOutbox < ActiveRecord::Migration[7.0]
+  def change
+    create_table :transactional_outbox_outboxes do |t|
+      t.string :aggregate, null: false
+      t.uuid :aggregate_identifier, null: false
+      t.string :event, null: false
+      t.uuid :identifier, null: false, index: true
+      t.jsonb :payload
+
+      t.timestamps
+    end
+  end
```

Comment:
Should you also drop the previous existing outbox tables? 😄

Reply:
we still need to discuss if we want to have 1 table or multiple tables for the outboxes